src/components/Timer/Timer.scss

```diff
@@ -0,0 +1,94 @@
+.timer {
+  position: fixed;
+  bottom: 16px;
+  left: 50%;
+  transform: translateX(-50%);
+  z-index: 300;
+}
+.timer--top {
+  bottom: auto;
+  top: 98px;
+}
+.timer > * {
+  color: white;
+  background-color: #0057ff;
+  vertical-align: bottom;
+}
+.timer:not(.timer--top) > * {
+  box-shadow: 0 8px 16px #0057ff25;
+}
+@media (prefers-color-scheme: dark) {
+  .timer > * {
+    background-color: #ff0069;
+  }
+  .timer:not(.timer--top) > * {
+    box-shadow: 0 8px 16px #ff006925;
+  }
+}
+.timer > span {
+  display: inline-block;
+  border-radius: 18px;
+  width: 84px;
+  cursor: default;
+  font-weight: bold;
+  line-height: 36px;
+  text-align: center;
+}
+.timer > span:not(:only-child) {
+  margin-right: 10px;
+}
+.timer > button {
+  border: none;
+  display: inline-block;
+  height: 36px;
+  width: 36px;
+  border-radius: 18px;
+  cursor: pointer;
+  padding: 0;
+}
```

Comment:
You can think about grouping all timer and child css styles into one `.timer`.
e.g.
```
.timer {
  > span { ... }
  > button { ... }
}
```

Reply:
I know but it's more legible if you don't nest anything imo

Reply:
Yes - this is a valid point 👍